pgx_async.opam

```diff
@@ -12,14 +12,10 @@ build: [
 ]
 
 depends: [
-  "ocaml" { >= "4.08.0" }
```

Comment:
why remove this?

Reply:
Because we already have that requirement in pgx.opam. Is there a reason we should duplicate it?